src/PackageCompiler.jl

```diff
@@ -760,7 +763,8 @@ function create_app(package_dir::String,
                     audit=true,
                     force=false,
                     c_driver_program::String=joinpath(@__DIR__, "embedding_wrapper.c"),
-                    cpu_target::String=default_app_cpu_target())
+                    cpu_target::String=default_app_cpu_target(),
+                    include_lazy_artifacts::Bool=true)
```

Comment:
I imagine I'm missing something, but at first blush it seems that this argument should be passed to the `_create_app` call in this method's body? :)

Reply:
I don't think you're missing anything--it definitely should be passed.

MR: https://github.com/JuliaLang/PackageCompiler.jl/pull/530